cmd/crc/cmd/daemon_windows.go

```diff
@@ -1,14 +1,86 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/code-ready/crc/pkg/crc/api"
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
 )
 
 func runDaemon() {
 	// Remove if an old socket is present
 	os.Remove(constants.DaemonSocketPath)
-	api.RunDaemonService("CodeReady Containers", false, newConfig, newMachineWithConfig)
+	RunDaemonService("CodeReady Containers", false)
+}
+
+var elog debug.Log
+
+type daemonService struct {
+	socketPath string
+}
+
+func (service *daemonService) Execute(args []string, r <-chan svc.ChangeRequest, changes chan<- svc.Status) (ssec bool, errno uint32) {
+	const cmdsAccepted = svc.AcceptStop | svc.AcceptShutdown
+	crcAPIServer, err := api.CreateServer(service.socketPath, newConfig, newMachineWithConfig)
```

Comment:
In last commit we removed `crc` from `APIServer` and  here we are again using it?